composer.json

```diff
@@ -24,20 +24,21 @@
         "php": "^7.2 || ^8.0",
         "jms/serializer": "^3.0",
         "jms/metadata": "^2.5",
-        "symfony/dependency-injection": "^3.3 || ^4.0 || ^5.0",
-        "symfony/framework-bundle": "^3.0 || ^4.0 || ^5.0"
+        "symfony/dependency-injection": "^3.3 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/framework-bundle": "^3.0 || ^4.0 || ^5.0 || ^6.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^8.1",
         "doctrine/orm": "^2.4",
         "phpunit/phpunit": "^8.0 || ^9.0",
-        "symfony/expression-language": "^3.0 || ^4.0 || ^5.0",
-        "symfony/finder": "^3.0 || ^4.0 || ^5.0",
-        "symfony/form": "^3.0 || ^4.0 || ^5.0",
-        "symfony/stopwatch": "^3.0 || ^4.0 || ^5.0",
-        "symfony/twig-bundle": "*",
-        "symfony/validator": "^3.0 || ^4.0 || ^5.0",
-        "symfony/yaml": "^3.0 || ^4.0 || ^5.0"
+        "symfony/expression-language": "^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/finder": "^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/form": "^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/stopwatch": "^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/templating": "^3.0 || ^4.0 || ^5.0 || ^6.0",
```

Comment:
The templating component wasn't being loaded in at all previously, so now it's included for dev and a basic test class added in.